code/rtsp_save_process/blur_process.py

```
@@ -0,0 +1,575 @@
+- 독립적으로 실행 가능한 프로세스
+"""
+
+import os
+import sys
+import time
+import json
+import signal
+import logging
+import argparse
+import cv2
+import multiprocessing
```

Ruff (0.12.2)

`multiprocessing` imported but unused

Remove unused import: `multiprocessing`

(F401)